src/csharp/Grpc.IntegrationTesting/TestGrpc.cs

```diff
@@ -34,7 +34,15 @@ public static partial class TestService
   {
     static readonly string __ServiceName = "grpc.testing.TestService";
 
-    static readonly grpc::Marshaller<global::Grpc.Testing.Empty> __Marshaller_grpc_testing_Empty = grpc::Marshallers.Create((arg) => global::Google.Protobuf.MessageExtensions.ToByteArray(arg), global::Grpc.Testing.Empty.Parser.ParseFrom);
+    static readonly grpc::Marshaller<global::Grpc.Testing.Empty> __Marshaller_grpc_testing_Empty = grpc::Marshallers.Create(
```

Comment:
This illustrates the output of the codegen which so far I've been doing manually for testing the new APIs.